Sources/App/Icon/DownloadIconCommand.swift

```diff
@@ -339,6 +339,11 @@ struct DownloadIconCommand: AsyncCommand {
                     return
                 }
                 
+                if v.variable == .convective_cloud_top || v.variable == .convective_cloud_base {
+                    // Icon sets points where no convective clouds are present to -500
+                    data.data = data.data.map { $0 < -450 ? .nan : $0 }
```

Comment:
`NaN` should only mark if no data is available for a given time-step. The database update process will then only update finite values into the final database. Therefore consecutive runs will never set convection from a finite convection height value to "no height". Although it is not ideal, but could you set it `0` instead of `NaN`?

Reply:
Thanks for your feedback and the explanation!

I wonder, given that `NaN` is reserved for data that is really missing, whether it would be the most reasonable decision to just leave the -500 values instead of setting them to 0? The motivation would be that there are some sub sea-level places on earth where a cloud base of 0 might actually lead to confusion.

Reply:
If `0` is used for both `convective_cloud_top` and `convective_cloud_bottom` does it correctly indicate "no convection"?

I use the same approach for cloud top and bottom for the DMI HARMONIE model: https://github.com/open-meteo/open-meteo/blob/e6aaea7dd0e5587f48f4f2d96a3ebd2f3f1dbb70/Sources/App/Dmi/DmiDownloader.swift#L214-L221

Reply:
It does (the most common blue tone represents a value of -500).
![Screenshot from 2024-12-03 10-31-03](https://github.com/user-attachments/assets/c7e60561-f5d6-4e0c-bb6a-7056478e80e7)

I also verified this for all time steps with a small python script. 

I will then change it in the downloader to 0.